internal/app/mapper/mapper.go

```diff
@@ -223,15 +230,18 @@ func getResultFromRequestNodeFragmentRule(fragmentRule *rule, node *core.Node) (
 		return false, "", nil
 	}
 
-	return getResultFromRequestNodePredicate(fragmentRule.GetResult(), node)
+	return getResultFromRequestNodePredicate(resultPredicate, node)
 }
 
-func getResultFromAndResultFragmentRule(fragmentRule *rule, node *core.Node) (bool, string, error) {
+func getResultFromAndResultFragmentRule(
+	fragmentRule *rule,
+	node *core.Node,
+	resourceNames []string) (bool, string, error) {
```

Comment:
I wish `gofmt` could handle formatting of function arguments.

Reply:
is there a different way you have in mind for function structure? I can try that out.